.github/workflows/update-sync-param-files.py

```diff
@@ -0,0 +1,42 @@
+import argparse
```

Comment:
Please explain more details about the algorithm in this Python file.

Reply:
I refactored the script.
Would you take a look at the latest one?

Reply:
Thank you! I'll take a look at it later.